src/main/kotlin/no/nav/familie/ef/sak/simulering/SimuleringController.kt

```diff
@@ -31,4 +31,12 @@ class SimuleringController(
         tilgangService.validerTilgangTilBehandling(saksbehandling, AuditLoggerEvent.UPDATE)
         return Ressurs.success(simuleringService.simuler(saksbehandling))
     }
+
+    @GetMapping("/resultatstatus/{behandlingId}")
```

Comment:
Jeg synes ikke resultatstatus er så beskrivende for hva denne koden faktisk gjør. Forslag: rename til `/simuleringsresultat-er-endret` eller noe mer beskrivende